Sources/C++Workshop/Cards.h

```diff
@@ -83,15 +83,15 @@ struct PMCard: public Card<Deck> {
     /// the opponent's deck that share an attack or defense stat with the opponent's card.
     void effect(Card<Deck>& opponent_card, Player<Deck>& player, Player<Deck>& opponent) const override {
         for (auto& card : player.deck()) {
-            card.get().attack += opponent_card.attack;
```

Comment:
Q: why do we have this `.get()` in the first place?

Reply:
Oh is it [this unique_ptr::get()](https://www.cplusplus.com/reference/memory/unique_ptr/get/) that we originally called?

Reply:
If not called then it will be the reference wrapper object instead of the card object?

Reply:
Oh yeah you're right @zoeli618 , it's [the reference_wrapper one](http://www.cplusplus.com/reference/functional/reference_wrapper/get/). Thanks.